src/scenes/Question/scenes/Edit/components/Tips/Tips.jsx

```diff
@@ -1,16 +1,93 @@
 import React from 'react'
 
-import Card, { CardText } from 'components/Card'
+import { default as TipsComponent } from 'components/Tips'
 
 const Tips = props => (
-  <Card {...props}>
-    <CardText>
-      <h3 style={{ textAlign: 'center', color: 'var(--primary-color)' }}>
-        Tips to write great questions on the FAQ
-      </h3>
-      --Need help--
-    </CardText>
-  </Card>
+  <TipsComponent {...props} uid="tips_question">
+    <h3>Tips to write great questions on the FAQ</h3>
+    <div style={{ textAlign: 'center' }}>
+      <ul style={{ textAlign: 'left' }}>
+        <li>
+          Do some searches before you can be sure your question has not already
```

Comment:
I suggest "Search the FAQ before you add your question to make sure it doesn't already exist." English is more verb-oriented than French.